doc/python_reference.rst

```diff
@@ -1116,6 +1116,7 @@ Classes:
    UnsupervisedSpatialFilter
    Vectorizer
    ReceptiveField
+   TimeDelayingRidge
```

Comment:
do we want to expose this model to the user or just have them use it as a part of the RF class? I'm happy either way